docs/core/install/runtime.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Install .NET Core runtime on Windows, Linux, and macOS - .NET Core
+description: Learn how to install .NET Core on Windows, Linux, and macOS. Discover the dependencies required to run .NET Core apps.
+author: thraka
+ms.author: adegeo
+ms.date: 11/06/2019
+ms.custom: "updateeachrelease"
+zone_pivot_groups: operating-systems-set-one
+---
+
+# Install the .NET Core Runtime
+
+In this article, you'll learn how to download and install the .NET Core runtime. The .NET Core runtime is used to run apps created with .NET Core.
+
+You can download and install .NET Core directly with one of the following links:
+
+- [.NET Core 3.1 Preview 3 downloads](https://dotnet.microsoft.com/download/dotnet-core/3.1)
+- [.NET Core 3.0 downloads](https://dotnet.microsoft.com/download/dotnet-core/3.0)
+- [.NET Core 2.2 downloads](https://dotnet.microsoft.com/download/dotnet-core/2.2)
+- [.NET Core 2.1 downloads](https://dotnet.microsoft.com/download/dotnet-core/2.1)
+
+## Install with an installer
+
+Both Windows and macOS have standalone installers that can be used to install the .NET Core 3.0 runtime.
+
+- Windows [x64 CPUs](https://dotnet.microsoft.com/download/thank-you/dotnet-runtime-3.0.0-windows-x64-installer) | [x32 CPUs](https://dotnet.microsoft.com/download/thank-you/dotnet-runtime-3.0.0-windows-x86-installer)
+- macOS [x64 CPUs](https://dotnet.microsoft.com/download/thank-you/dotnet-runtime-3.0.0-macos-x64-installer)
```

Comment:
These are direct version links, which seems like a bad idea because it'll prevent people from getting security patches. It's already out of date vs. 3.0.1.

Here, `dotnet-runtime-3.0.0-macos-x64-installer`, line above has more 3.0.0 links. Same problem on the SDK install page.

I see this was discussed a bit in some earlier PR comments, is there some issue tracking this?

Reply:
Generally we're just tracking things in this PR but with over 200 comments its hard to track.